jcalapi/app.py

```diff
@@ -194,8 +214,12 @@ async def reload_exchange(
     backend = "exchange"
 
     LOGGER.info(
-        f"Fetch calendar events from Exchange for user {exchange_username}"
+        "Fetch calendar events from Exchange for user %s", exchange_username
```

Comment:
What did you do to my f-string? :cry:

Reply:
Resolved (https://github.com/pschmitt/jcalapi/pull/264/commits/2770d48b6d23e585c9d098003dfc6e3b9e11fcfa). I suppose it's my lint who change this based on that : https://pylint.readthedocs.io/en/stable/user_guide/messages/warning/logging-fstring-interpolation.html